testsuite/classlibrary/TestServer.sc

```diff
@@ -38,14 +33,1297 @@ TestServer : UnitTest {
 		this.sync;
 	}
 
+	tearDown {
+		server.quit;
```

Comment:
… and here.

Reply:
I'll remove this one, and all the ones that are after calls to `this.sync`.